en/config/autotune.md

```diff
@@ -23,7 +24,7 @@ This test ensures that the vehicle can fly safely in position controlled modes.
 
 :::note
 During [Airframe Setup](../config/airframe.md) you should have selected the frame that most closely matches your vehicle.
-This will usually be tuned well enough to fly, and it _may_ also be sufficiently well tuned to run autotuning.
+This may fly well enough to run autotuning.
```

Comment:
Changed, as most airframes that people will select now are generic, so less likely good enough to fly "out of the box". Not sure what people are supposed to do in this case.

Reply:
I would generally encourage people to start the bringup process with stock tunings and immediately autotune after a normal takeoff in manual mode. I know this isn't mainstream PX4 workflow, but this process has yet to fail me.

Reply:
@bresch 

- How risky would it be to advise this? I.e. use a stock tuning "generic X quad", "generic standard VTOL", configure, take off, and just run autotune without the pre-test? 
- What do you do if you can't get off the ground in the first place - i.e the stock tuning doesn't allow that? Does that mean control allocation is wrong?

@chris1seto Thanks. I suspect it does work in most cases. But then you have experience in brining up sane frames. I'm trying to mitigate risk for newbies who might not try again if their vehicle dies.